content/en/docs/concepts/networking/function-entrypoints.md

```diff
@@ -53,13 +52,32 @@ export FUNC_PATH=$(kubectl get function function-sample -o=jsonpath='{.status.ro
 
 Access `Function` using curl directly:
 ```shell
-curl -sv -HHOST:$FUNC_HOST http://$IP:$PORT$FUNC_PATH
+curl -sv -HHOST:$FUNC_HOST http://$IP$FUNC_PATH
 ```
 
-### Access by domain defined in OpenFunction Gateway
+### Access functions by the external address
+
+#### Configure DNS
+You have two options to configure DNS:
+
+- Option 1: Real DNS
+
+    The real DNS configuration is based on the network topology of your environment.
+
+- Option 2: Configure the local hosts file
 
-> You should configure your real DNS first, the relevant configuration is based on the network topology of your environment.
+    Get Function's HOSTNAME and Gateway's ip:
+    ```shell
+    export HOSTNAME="function-sample.default.ofn.io"
```

Comment:
Can this be a wildcard hostname？
    export HOSTNAME="*.default.ofn.io"

Reply:
hosts file not supported wildcard hostname. 

The documentation of dnsmasq and chrome pac may be added to the FAQ or Operations section later.

Reply:
ok